package.json

```diff
@@ -21,8 +21,8 @@
     "generate-style-spec": "node --no-warnings --loader ts-node/esm build/generate-style-spec.ts",
     "generate-typings": "node --no-warnings --loader ts-node/esm build/generate-typings.ts",
     "generate-docs": "node ${WATCH+--watch} --no-warnings --loader ts-node/esm build/generate-docs.ts",
-    "mkdocs": "docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material",
-    "mkdocs-build": "npm run generate-docs && docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material build",
+    "mkdocs": "docker build -t squidfunk/mkdocs-material . && docker run --rm -it -p 8000:8000 -v ${PWD}:/docs -e ENABLED_HTMLPROOFER=false squidfunk/mkdocs-material",
```

Comment:
Doesn't look like the plugin is disabled in local run.

Reply:
`-e ENABLED_HTMLPROOFER=false`

Reply:
Why not simply remove the docker image build in local run?

Reply:
Because even though the plugin is disabled, it still needs to be present because it is in the config.